crmsh/corosync.py

```diff
@@ -584,9 +587,39 @@ def create_configuration(clustername="hacluster",
         if transport != "udpu":
             ipv6_nodeid = "nodeid:  %d" % nodeid
 
+    quorum_tmpl = """quorum {
+    # Enable and configure quorum subsystem (default: off)
+    # see also corosync.conf.5 and votequorum.5
+    provider: corosync_votequorum
+    expected_votes: 1
+    two_node: 0
+}
+"""
+    if qdevice is not None:
+        quorum_tmpl = """quorum {
+    # Enable and configure quorum subsystem (default: off)
+    # see also corosync.conf.5 and votequorum.5
+    provider: corosync_votequorum
+    expected_votes: 1
+    two_node: 0
+    device {
+      votes: 1
+      model: net
+      net {
+        tls: off
+        host: %(ip)s
+        port: %(port)s
+        algorithm: %(algo)s
+        tie_breaker: %(tie_breaker)s
```

Comment:
the tie_breaker should be:

auto_tie_breaker: 1
auto_tie_breaker_node: lowest

auto_tie_breaker = 1 mean to enable auto_tie_breaker, this option is disabled by default, we should enable it.
while auto_tie_breaker_node is the preference that qnetd to decide who should have the votes if there is brain split